### ISSUE TITLE
add model to schemaSpec

### ISSUE THREAD

**Issue by tuurma** [TEIC/TEI#1580]
to accommodate default models or models targeting text nodes, etc

**Comment by cmsmcq** [comment]
The nature of the issue is not clear to this reader, nor the nature of the proposed solution.  Could they be expounded a bit more?

**Comment by jamescummings** [comment]
If I am understanding correctly the proposal is that the `<model>` element should be allowed directly inside `<schemaSpec>`. This is entirely parallel to when we allowed `<constraint>` directly inside `<schemaSpec>` to enable it to have default schematron rules that weren't localised to a particular element, etc.  I'm assuming that this would be for the same kind of thing but with the processing model behaviour.

**Comment by martinascholger** [comment]
F2F subgroup (Victoria) agreed and marked this as green

**Comment by sydb** [comment]
But, @jamescummings , we don’t allow `<constraint>` directly inside `<schemaSpec>`, only `<constraintSpec>`. Yes, I know that’s what you meant, but my point is that maybe we should hold off for 10 mins on this ticket and come up with a proper wrapper for `<model>`, `<modelSequence>`, and `<modelGrp>`, and allow **that** as a child of `<schemaSpec>`. I think `<models>`, `<modelSpec>`, `<modelDesc>`, and `<processing>` are all viable names for this wrapper.

**Comment by jamescummings** [comment]
@Sydb yes. constraintSpec. As I said. But yes. I agree with you.  From YVR.

**Comment by tuurma** [comment]
Sorry to get back to it so late. Are we happy to go with `modelSpec` as the wrapper then?

**Comment by ebeshero** [comment]
Should we think about this ticket in connection with https://github.com/TEIC/TEI/issues/1500 ? @hcayless @tuurma
